frontend/app/src/pages/defi/airdrops/index.vue

```diff
@@ -219,8 +246,26 @@ onMounted(async () => {
           />
           <span v-else>{{ row.details.length }}</span>
         </template>
-        <template #item.claimed="{ row: { claimed } }">
+        <template #item.claimed="{ row: { claimed, missingDecoder } }">
+          <RuiTooltip
+            v-if="missingDecoder"
+            :popper="{ placement: 'top' }"
+            :open-delay="400"
+            tooltip-class="max-w-[12rem]"
```

Comment:
The width for tooltip, doesn't need to be 100% precise, we can round it to simplify computation

Reply:
okay, will add it on another PR.